homeassistant/components/denonavr/media_player.py

```diff
@@ -338,6 +338,8 @@ def media_next_track(self):
 
     def select_source(self, source):
         """Select input source."""
+        if self._state != STATE_ON:
```

Comment:
So we have to check the state?
This is a polling integration, thus the state might not be up 2 date. Does it do harm to send a turn on, when already on?

Reply:
`turn_on` is just a single HTTP call which is pretty fast and does nothing if the AVR is already on.
I have removed the state check, PTAL?